google/cloud/bigquery/job/base.py

```diff
@@ -913,7 +924,7 @@ def _set_future_result(self):
     def done(
         self,
         retry: "retries.Retry" = DEFAULT_RETRY,
-        timeout: Optional[float] = None,
+        timeout: Optional[Union[float, object]] = POLLING_DEFAULT_VALUE,
```

Comment:
Same here. I don't think this needs to handle `POLLING_DEFAULT_VALUE` since it's not waiting for the job to finish, just making at most 1 API call.

Reply:
done